HasDropDown.js

```diff
@@ -225,7 +225,7 @@ define([
 		postRender: function () {
 			this.behaviorNode = this.behaviorNode || this;
 			this.buttonNode = this.buttonNode || this.behaviorNode;
-			this.popupStateNode = this.focusNode || this.buttonNode;
+			this.popupStateNode = this.popupNode || this.focusNode || this.buttonNode;
```

Comment:
@wkeese What about the default value for `popupStateNode`? I was thinking about doing this:
```
this.popupStateNode = this.popupStateNode || this.focusNode || this.buttonNode;
```
Mainly to be backward compatible.